app/resources/api/rest/customer/v1/cdr_resource.rb

```diff
@@ -0,0 +1,56 @@
+class Api::Rest::Customer::V1::CdrResource < BaseResource
+  model_name 'Cdr::Cdr'
+
+  key_type :uuid
+  primary_key :uuid
+
+  attributes :time_start,
+             :time_connect,
+             :time_end,
+             :duration,
+             :success, # TODO: or it should be "status"?
+             :destination_initial_interval,
+             :destination_initial_rate,
+             :destination_next_interval,
+             :destination_next_rate,
+             :destination_fee,
+             :customer_price,
+             :src_name_in,
+             :src_prefix_in,
+             :from_domain,
+             :dst_prefix_in,
+             :to_domain,
+             :ruri_domain,
+             :diversion_in,
+             :local_tag,
+             :lega_disconnect_code, # TODO: or it shold be "leg_a_dc"
+             :lega_disconnect_reason, # TODO: or it should be "leg_a_reason"
+             :lega_rx_payloads,
+             :lega_tx_payloads,
+             :auth_orig_transport_protocol_id, # TODO: relation "auth_orig_transport_protocol"?
```

Comment:
@dmitry-sinina perhaps it should be replaced with relation `auth_orig_transport_protocol`?

Reply:
yes